From bac9c23e607bf4264b1139f675a19529001980cb Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:25:35 -0400
Subject: [tools] perf: Align perf version output to other build messages

Message-id: <1408462094-14194-167-git-send-email-jolsa@redhat.com>
Patchwork-id: 88206
O-Subject: [PATCH RHEL7.1 BZ1131394 166/325] perf tools: Align perf version output to other build messages
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 3fae82db558468c01f36eb6398e9459ac240e697
Author: Ingo Molnar <mingo@kernel.org>
Date: Wed Oct 9 11:49:28 2013 +0200

description
===========
Before:

  CC util/pmu.o
  CC util/parse-events.o
PERF_VERSION = 3.12.rc4.g1b30c
  CC util/parse-events-flex.o
  GEN perf-archive

After:

  CC util/pmu.o
  CC util/parse-events.o
  PERF_VERSION = 3.12.rc4.g1b30c
  CC util/parse-events-flex.o
  GEN perf-archive
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/PERF-VERSION-GEN b/tools/perf/util/PERF-VERSION-GEN
index 15a77b7..ce7a804 100755
--- a/tools/perf/util/PERF-VERSION-GEN
+++ b/tools/perf/util/PERF-VERSION-GEN
@@ -40,7 +40,7 @@ else
  VC=unset
 fi
 test "$VN" = "$VC" || {
- echo >&2 "PERF_VERSION = $VN"
+ echo >&2 "  PERF_VERSION = $VN"
  echo "#define PERF_VERSION \"$VN\"" >$GVF
 }
 
-- 
1.7.1